src/kubernetes-ingress-controller/references/version-compatibility.md

```diff
@@ -58,6 +64,31 @@ other enterprise functionality, built on top of the Open-Source {{site.base_gate
 | Kubernetes 1.24           | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
 | Kubernetes 1.25           | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
 
+### Gateway API
+
+The following table presents the compatibility of {{site.kic_product_name}}'s [Gateway API][gateway-api]
+with specific Kubernetes minor versions. As {{site.kic_product_name}} implements Gateway API features in addition to the upstream
+project, which defines [its own compatibility declarations][gateway-api-supported-versions], the expected compatibility
+of Gateway API features might be limited to those.
```

Comment:
@cloudjumpercat I think I misled you here by writing the "implements these features on top of the upstream project" part. What I meant is more "implements these features using the upstream project". "in addition to" is not exactly this. 

```suggestion
with specific Kubernetes minor versions. As {{site.kic_product_name}} implements Gateway API features using the upstream
project, which defines [its own compatibility declarations][gateway-api-supported-versions], the expected compatibility
of Gateway API features might be limited to those.
```